docs/source/setup_osx.rst

```diff
@@ -117,6 +118,32 @@ default use password `seedpass` when prompted
     psql -c 'ALTER USER seeduser CREATEDB;'
     psql -c 'ALTER USER seeduser CREATEROLE;'
 
+
+PostGIS 2.5
```

Comment:
I have more confidence in the single Homebrew command as I installed PostGIS that way. I'm less confident in the MacPorts install as I haven't used MacPorts prior. 

If no one here reaches out with corrections/feedback from their setup, I'll likely have someone else run through this portion of the setup for feedback.

Reply:
looks great to me, we can see if nrel team has any thoughts.